src/features/events/layout/EventLayout.tsx

```diff
@@ -27,11 +33,102 @@ const EventLayout: React.FC<SurveyLayoutProps> = ({
     (env) => new EventDataModel(env, parseInt(orgId), parseInt(eventId))
   );
 
+  const showTimeOnly = (value: Date) => {
+    return value.toLocaleTimeString('en-Us', {
+      hour: '2-digit',
+      hour12: false,
+      minute: '2-digit',
+      timeZone: 'UTC',
+    });
+  };
+
   return (
     <TabbedLayout
       baseHref={`/organize/${orgId}/projects/${campaignId}/events/${eventId}`}
       defaultTab="/"
-      subtitle={<Box alignItems="center" display="flex"></Box>}
+      subtitle={
+        <Box alignItems="center" display="flex">
+          <EventStatusChip state={model.state} />
+          <Box marginX={2}>
+            <ZUIFuture future={model.getData()}>
+              {(data) => {
+                const startDate = new Date(data.start_time);
+                const startTime = showTimeOnly(startDate);
+
+                const endDate = new Date(data.end_time);
+                const endTime = showTimeOnly(endDate);
+
+                const isOnSameDay = startDate.getDate() === endDate.getDate();
+
+                const isToday =
+                  startDate.toDateString() === new Date().toDateString();
+
+                const labels: ZUIIconLabelProps[] = [];
+
+                if (startDate && endDate) {
+                  labels.push({
+                    icon: <EventIcon />,
+                    label: (
+                      <>
+                        {isToday && (
+                          <>
+                            <Msg id={messageIds.stats.today} />
+                            <Msg
+                              id={messageIds.stats.todaysEventTime}
+                              values={{ end: endTime, open: startTime }}
+                            />
+                          </>
+                        )}
+                        {!isToday && (
+                          <Msg
+                            id={messageIds.stats.eventTime}
+                            values={{
+                              end: `${!isOnSameDay ? ',' : ''} ${endTime}`,
```

Comment:
Here you are making localization decisions (to use "comma") in the program code. Such decisions should always be made in the localized messages, because they may differ between different languages.

I would recommend changing this logic so that you have separate messages for events depending on when they happen and whether start and end are on the same day, i.e.:

* Starts and ends on same day (that is not today), `singleDay`
* Starts and ends on different days (not today), `multiDay`
* Starts today, ends today, e.g.  `singleDayToday`
* Starts today, ends later, e.g. `multiDayToday`